### DISCUSSION TITLE
Periodic hello.c compilation

### DISCUSSION THREAD

**Discussion by atalii** [KSmanis/docker-gentoo-distcc#41] [Q&A] [answered]
Hello! I'm testing this out to use on my home server, and I see the following log line indicating that the server keeps compiling what I assume is the sample program in the tests dir:

```
distccd[8] (dcc_job_summary) client: 127.0.0.1:46724 COMPILE_OK exit:0 sig:0 core:0 ret:0 time:50ms gcc hello.c
```

I'd rather not compile hello world every so often; is this behavior intentional? If so, is there a way to disable it? Thanks so much! This is a very helpful resource :)

**Comment by KSmanis** [comment, marked as answer]
Hi, you just noticed the container's [HEALTHCHECK](https://github.com/KSmanis/docker-gentoo-distcc/blob/aa0b40f5f1dc5be0ceda5348213460d211e2b2c4/Dockerfile#L41). Depending on how you ran the container, you can turn it off either by passing the [`--no-healthcheck`](https://docs.docker.com/engine/reference/run/#healthcheck) flag to `docker run` or [`healthcheck: {disable: true}`](https://docs.docker.com/compose/compose-file/05-services/#healthcheck) in docker-compose. You may also manually adjust the default interval if you prefer to leave it on.
